ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -16,7 +16,15 @@
                     <div>
                         @if (@Model.ShowLink)
                         {
-                            <a href="@Model.RedirectPath" class="notification-banner-id-link" id="notification-banner-id">#@Model.NotificationId</a>
+                            @if (Model.NotificationStatus == NotificationStatus.Legacy)
+                            {
+                                var legacyId = Model.LTBRID ?? Model.ETSID;
+                                <a href="@Model.RedirectPath" class="notification-banner-id-link" id="notification-banner-id">#@legacyId</a>

```

Comment:
I think we want to make it clear where this came from, so something like `ETS #@legacyId` or `LTBR #@legacyId`. There will be cases with both - perhaps that's worth accounting for, too?